### PR TITLE
Add recipe for maup

### PR DESCRIPTION
**Pull request by maxhully**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/maup/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (chrisburr) [inline recipes/maup/meta.yaml]: Can you remove the comments that are inherited from the example?

File: recipes/maup/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - python >= 3.6
```
Comment (chrisburr) [inline recipes/maup/meta.yaml]: This needs to be formatted like this to be valid:
```suggestion
    - python >=3.6
```

File: recipes/maup/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    - pip
+  run:
+    - pandas
+    - shapely
+    - geopandas
+    - python >= 3.6
```
Comment (chrisburr) [inline recipes/maup/meta.yaml]: Similarly:
```suggestion
    - python >=3.6
```

File: recipes/maup/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - maup
+
+about:
+  home: http://github.com/simplejson/simplejson
```
Comment (chrisburr) [inline recipes/maup/meta.yaml]: This should be changed

File: recipes/maup/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (chrisburr) [inline recipes/maup/meta.yaml]: This can be simplified now:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```

**Comment by maxhully** [comment]
I committed all of those changes. Thanks!

**Pull request merged**